Plugins/Locales/koKR.lua

```diff
@@ -150,18 +150,18 @@ L.audio = "음성"
 L.music = "배경음악"
 L.ambience = "환경 소리"
 L.sfx = "효과음"
---L.disableMusic = "Mute music (recommended)"
---L.disableAmbience = "Mute ambient sounds (recommended)"
---L.disableSfx = "Mute sound effects (not recommended)"
---L.disableAudioDesc = "The '%s' option in WoW's sound options will be turned off, then turned back on when the boss encounter is over. This can help you focus on warning sounds from BigWigs."
---L.blockTooltipQuests = "Block tooltip quest objectives"
---L.blockTooltipQuestsDesc = "When you need to kill a boss for a quest, it will usually show as '0/1 complete' in the tooltip when you place your mouse over the boss. This will be hidden whilst in combat with that boss to prevent the tooltip growing very large."
---L.blockObjectiveTracker = "Hide quest tracker"
---L.blockObjectiveTrackerDesc = "The quest objective tracker will be hidden during a boss encounter to clear up screen space.\n\nThis will NOT happen if you are in a mythic+ or are tracking an achievement."
+L.disableMusic = "배경음악 끄기 (권장)"
+L.disableAmbience = "환경 소리 끄기 (권장)"
+L.disableSfx = "효과음 끄기 (권장하지 않음)"
+L.disableAudioDesc = "WoW의 소리 설정에서 '%s' 설정이 꺼지고 교전이 끝난 후에 다시 켜집니다. BigWigs의 알림 소리에 더 집중하는데 도움이 됩니다."
+L.blockTooltipQuests = "툴팁에서 퀘스트 목표 표시를 숨깁니다."
+L.blockTooltipQuestsDesc = "우두머리를 처치하는 퀘스트를 진행 중인 경우 해당 우두머리에 마우스를 올렸을 때 보통 '0/1 완료' 같은 문구가 툴팁에 표시됩니다. 전투 중에 이 문구들을 숨김으로써 툴팁의 크기가 너무 크게 확장되는 것을 방지합니다."
+L.blockObjectiveTracker = "퀘스트 추적기 숨김"
+L.blockObjectiveTrackerDesc = "보스 전투 중에 퀘스트 추적기를 숨겨서 화면의여유 공간을 넓힙니다. 신화 쐐기돌이나 업적을 추적 중일 때는 숨기지 않습니다."
```

Comment:
You removed `\n\n` from this translation, this is a special code that creates a new line. It must stay.

Reply:
Valid translation would be:
```L.blockObjectiveTrackerDesc = "보스 전투 중에 퀘스트 추적기를 숨겨서 화면의 여유 공간을 넓힙니다.\n\n신화 쐐기돌이나 업적을 추적 중일 때는 숨기지 않습니다."```

But I don't know hot to fix this. Should I make another pull request? Sorry I am not familiar with these things.

Reply:
@mundi4 If you click on the "Files Changed" tab at the top, then click the 3 "..." one of the options should be edit file. I'll fix this one.